Update Canal issues with IP exhaustion (#3039)
rancher/rke2: docs/known_issues.md

net.ipv4.conf.all.forwarding=1
net.ipv6.conf.all.forwarding=1
```

## Canal and IP exhaustion

> By default Canal keeps track of pod IPs by creatig a lock file for each IP in `/var/lib/cni/networks/k8s-pod-network`. Each IP belongs to a single pod and will be deleted as soon as the pod is removed. However, in the unlikely event that containerd loses track of the running pods, lock files may be leaked and Canal will not be able to reuse those IPs anymore. If this occurs, you may experience IP exhaustion errors, for example:

Reviewer: nit: typo
```suggestion
By default Canal keeps track of pod IPs by creating a lock file for each IP in `/var/lib/cni/networks/k8s-pod-network`. Each IP belongs to a single pod and will be deleted as soon as the pod is removed. However, in the unlikely event that containerd loses track of the running pods, lock files may be leaked and Canal will not be able to reuse those IPs anymore. If this occurs, you may experience IP exhaustion errors, for example:
```

Author: good catch

> ```bash

Reviewer: nit: console or just no syntax for log messages; bash will highlight weird stuff if you use it for log output.
```suggestion
```console
```

Author: aaaaah ok, thanks!

net.ipv4.conf.all.forwarding=1
net.ipv6.conf.all.forwarding=1
```

## Canal and IP exhaustion

> By default Canal keeps track of the used IPs by saving them in `/var/lib/cni/networks/k8s-pod-network`. Each IP belongs to a pod and will be deleted as soon as the pod is removed. However, in the unlikely event that containerd loses track of the running pods, there might be IPs which are left over. Canal will not be able to reuse those IPs anymore and you could experience IP exhaustion errors. In such cases, please remove the left over IPs from that directory and report the problem, specifying how it was triggered

Reviewer: Can we quote the specific error message that users will see when IPs are exhausted?

Couple nits on the spelling and punctuation.
```suggestion
By default Canal keeps track of pod IPs by creating a lock file for each IP in `/var/lib/cni/networks/k8s-pod-network`. Each IP belongs to a single pod and will be deleted as soon as the pod is removed. However, in the unlikely event that containerd loses track of the running pods, lock files may be leaked, and Canal will not be able to reuse those IPs anymore. If this occurs, you may experience IP exhaustion errors. To resolve this, you can manually remove unused IPs from that directory. If you need to do this, please report the problem via GitHub, making sure to specify how it was triggered.
```

Author: Thanks, done!
